sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -665,14 +665,14 @@ export class CryptographyClient {
    * // or
    * let client = new CryptographyClient(keyVaultKey, credentials);
    * ```
-   * @param key The key to use during cryptography tasks.
+   * @param key The key to use during cryptography tasks. It can be either a Key Vault Key identifier (or URL) or a full KeyVaultKey.
```

Comment:
Is "Identifier" and "URL" different things?
Or is Keyvault id itself an url?

Reply:
the KeyVault identifier is itself a URL

Reply:
Interesting, when I see "It can be either A or B", I take it to mean that they are two different things and either would work.

Between "identitifer" and "url", which term is used more in the service documentation and in the portal?
Are we using "identifier", because in the SDK, the corresponding property name is "id"?

Reply:
Yes, exactly. The "key ID" of a Key Vault Key is indeed a URL. Yes, it's confusing 😅

Reply:
Ok, I read through the docs in https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates. It indeed is a url!! 

How about this:

```suggestion
   * @param key The key to use during cryptography tasks. You can also pass the identifier of the key i.e its url here.
```